engine/api/event/publish_workflow_run.go

```diff
@@ -6,15 +6,27 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/go-gorp/gorp"
-
 	"github.com/ovh/cds/sdk"
 	"github.com/ovh/cds/sdk/log"
 )
 
-func publishRunWorkflow(ctx context.Context, payload interface{}, key, workflowName, appName, pipName, envName string, num int64, sub int64, status string, tags []sdk.WorkflowRunTag, eventIntegrations []sdk.ProjectIntegration) {
-	eventIntegrationsID := make([]int64, len(eventIntegrations))
-	for i, eventIntegration := range eventIntegrations {
+type publishWorkflowRunData struct {
+	projectKey        string
+	workflowName      string
+	applicationName   string
```

Comment:
i think applicationName, environmentName, pipelineName and EventIntegrationsID  doesn't have sens in this event